webapp/package.json

```
@@ -2,8 +2,8 @@
   "name": "@mattermost/webapp",
   "private": true,
   "engines": {
-    "node": "^18.10.0",
-    "npm": "^9.0.0 || ^10.0.0"
+    "node": "^20.11.0",
+    "npm": "^9.0.0 || ^10.2.0"
```

Could you change these to be a bit less restrictive? Until we know there's an issue with older versions, I don't think we need to force people off the old ones
```suggestion
    "node": "^18.10.0 || ^20.0.0",
    "npm": "^9.0.0 || ^10.0.0"
```

---

Actually, I have another idea. We just had someone run into an issue with using Node 21 which, as far as I know, still works with MM. Let's try making not blocking newer versions of Node until we find out that there's an issue with them

```suggestion
    "node": ">18.10.0",
    "npm": "^9.0.0 || ^10.0.0"
```

---

changes done